docs/application-connector/docs/001-overview-application-connector.md

```diff
@@ -2,11 +2,11 @@
 title: Overview
 ---
 
-The Application Connector (AC) is a proprietary Kyma implementation that allows you to connect with external solutions. No matter if you want to integrate an on-premise or a cloud system, the integration process doesn't change, which allows to avoid any configuration and network-related problems.
+The Application Connector (AC) is a proprietary Kyma implementation that allows you to connect with external solutions. No matter if you want to integrate an on-premise or a cloud system, the integration process doesn't change, which allows to avoid any configuration or network-related problems.
```

Comment:
doesn't > does not

Reply:
No need to be so strict about contractions. We allow them where they sound natural, that's the consensus ;)

Reply:
Sure, got it :)